Formula/goreman.rb

```diff
@@ -0,0 +1,50 @@
+require "language/go"
+
+class Goreman < Formula
+  desc "Foreman clone written in Go"
+  homepage "https://github.com/mattn/goreman"
+  url "https://github.com/mattn/goreman/archive/v0.1.1.tar.gz"
+  sha256 "d234ae81a397ab04338f26e30db79811715202413e0a3b9ff20395df7df675df"
+
+  bottle do
+    cellar :any_skip_relocation
+    sha256 "" => :mojave
+    sha256 "" => :high_sierra
+    sha256 "" => :sierra
+    sha256 "" => :el_capitan
+  end
+
+  depends_on "go"
+
+  go_resource "github.com/joho/godotenv" do
```

Comment:
`go_resource` are deprecated and are not acceptable in new formulae, please ask upstream to implement go vendoring.